src/lib/plugin.js

```diff
@@ -25,6 +25,7 @@ const defaultConnectTimeout = 60000
 const wsReconnectDelayMin = 10
 const wsReconnectDelayMax = 500
 const defaultMessageTimeout = 5000
+const authTokenMaxAge = 7 * 24 * 60 * 60 * 1000 // one week
```

Comment:
I don't think so. The token is only checked once, when the websocket first connects.